examples/twittermap/web/app/views/twittermap/main.scala.html

```diff
@@ -69,6 +69,8 @@
   <script src='@routes.Assets.versioned("lib/angular-leaflet-directive/angular-leaflet-directive.js")'></script>
   <script src='@routes.Assets.versioned("lib/jquery/jquery.min.js")'></script>
   <script src='@routes.Assets.versioned("lib/hashmap/hashmap.js")'></script>
+  <script src = "http://d3js.org/d3.v4.min.js" charset="utf-8"></script>
+  <script> d3version4 = d3; window.d3 = null; </script>
```

Comment:
Can we move this part of the script outside of `main.scala.html` file? It will make more sense that put them into a controller.js file.